TASK/TaskGoToCoordAnyMeansExtraParams.md

```diff
@@ -5,24 +5,104 @@ ns: TASK
 
 ```c
 // 0x1DD45F9ECFDB1BC9 0x094B75EF
-void TASK_GO_TO_COORD_ANY_MEANS_EXTRA_PARAMS(Ped ped, float x, float y, float z, float speed, Any p5, BOOL p6, int walkingStyle, float p8, Any p9, Any p10, Any p11);
+void TASK_GO_TO_COORD_ANY_MEANS_EXTRA_PARAMS(Ped ped, float x, float y, float z, float fMoveBlendRatio, cs_type(Any) Vehicle vehicle, BOOL bUseLongRangeVehiclePathing, int drivingFlags, float fMaxRangeToShootTargets, cs_type(Any) float fExtraVehToTargetDistToPreferVeh, cs_type(Any) float fDriveStraightLineDistance, cs_type(Any) int iExtraFlags, float fWarpTimerMS);
 ```
 
-```
-NativeDB Added Parameter 13: Any p12
+Same as `TASK_GO_TO_COORD_ANY_MEANS` but with extra parameters.
+
+```c
+enum eDrivingMode {
```

Comment:
Let's avoid duplicating information, delete this `enum` and just tell them to refer to the enum located at [`TASK_GO_TO_COORD_ANY_MEANS`](#_0x5BC448CB78FA3E88). That way the enum is centralized and if it needs to be changed in the future, there's only one place where it needs to be changed.

For example: 

See `eDrivingMode` enum in the following native [`TASK_GO_TO_COORD_ANY_MEANS`](#_0x5BC448CB78FA3E88).